makefiles/libc/picolibc.mk

```diff
@@ -6,6 +6,11 @@ ifneq (,$(filter picolibc,$(USEMODULE)))
         CFLAGS += -fshort-wchar
         LINKFLAGS += -Wl,--no-wchar-size-warning
     endif
+  else
+    $(warning picolibc was selected to be build but no picolibc.spec could be found)
+    $(warning you might want to install "picolibc" for "$(TARGET_ARCH)")
+    $(warning or add "FEATURES_BLACKLIST += picolibc" to Makefile)
+    $(error   check your installation or build configuration.)
```

Comment:
This error shows up even when not building, I think that is not desired and is causing `compile_and_test_for_boards` to fail: `https://github.com/RIOT-OS/RIOT/runs/2000318630?check_suite_focus=true`. I think we need to adapt this so that is only called if `all` or any building target is somehow called.